AppSandboxFileAccess/Classes/AppSandboxFileAccess.m

```diff
@@ -127,9 +127,15 @@ - (NSData *)persistPermissionURL:(NSURL *)url {
 	NSParameterAssert(url);
 	
 	// store the sandbox permissions
-	NSData *bookmarkData = [url bookmarkDataWithOptions:NSURLBookmarkCreationWithSecurityScope includingResourceValuesForKeys:nil relativeToURL:nil error:NULL];
+	NSData *bookmarkData = [url bookmarkDataWithOptions:NSURLBookmarkCreationWithSecurityScope
+						 includingResourceValuesForKeys:nil
+										  relativeToURL:nil
+												  error:NULL];
 	if (bookmarkData) {
-		[AppSandboxFileAccessPersist setBookmarkData:bookmarkData forURL:url];
+		if (self.bookmarkPersistanceDelegate)
+			[self.bookmarkPersistanceDelegate setBookmarkData:bookmarkData forURL:url];
+		else
+			[AppSandboxFileAccessPersist setBookmarkData:bookmarkData forURL:url];
```

Comment:
Could you add in `{` and `}` on these lines to make it consistent with the style of the existing code?